src/burn/drv/pre90s/d_armedf.cpp

```diff
@@ -1260,7 +1260,7 @@ static void draw_sprites(INT32 priority)
 		sy -= yoffset;
 		sx -= xoffset;
 
-		if (sx < -15 || sy < -15 || sx >= nScreenWidth || sy >= nScreenHeight) continue;
+		if (code > 0x7ff || sx < -15 || sy < -15 || sx >= nScreenWidth || sy >= nScreenHeight) continue;
 

```

Comment:
Hi,
Instead of checking code like that, how about fix the bitmask:
		code       &= 0xfff;
change to 
		code       &= 0x7ff;

sometimes upper bits of sprite code is used for internal game stuff, masking it will give the correct code.

best regards,
- dink

Reply:
I checked the source of latest MAME and confirmed that only changing the mask to 0x7ff was fine.
I committed the change it.
Thanks for review.

Reply:
welcome :) Looks good!